core/src/main/java/jenkins/websocket/WebSocketSession.java

```diff
@@ -108,8 +108,8 @@ protected final Future<Void> sendBinary(ByteBuffer data) throws IOException {
         return handler.sendBinary(data);
     }
 
-    protected final void sendBinary(ByteBuffer partialByte, boolean isLast) throws IOException {
-        handler.sendBinary(partialByte, isLast);
+    protected final Future<Void> sendBinary(ByteBuffer partialByte, boolean isLast) throws IOException {
+        return handler.sendBinary(partialByte, isLast);
```

Comment:
Unused method, so OK to break binary compatibility here. Better to do it now and clean up the API while it is unused rather than to get stuck with a bad API that we can never change once it gets used for the first time.